frontends/packages/livnote/desktop/components/notes/commentsService.ts

```
@@ -0,0 +1,307 @@
+  Comment,
+  CommentThread,
+  CommentPosition,
+  CreateCommentParams,
+  UpdateCommentParams,
+  CommentEvent,
+  UserInfo
+} from "../../types/notes.types";
+
+export class CommentsService {
+  private commentsMap: Y.Map<CommentThread>;
+  private callbacks: Map<string, Function[]> = new Map();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)